lib/space.rb

```diff
@@ -1,2 +1,17 @@
+require 'pg'
+
 class Space
-end
\ No newline at end of file
+  # attr_reader :spaces, :name, :description
```

Comment:
Let's remove these comments before merging.  We might need this code later but it's easy to add back in.